packages/react-core/src/components/Menu/Menu.tsx

```diff
@@ -63,6 +63,8 @@ export interface MenuProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'r
   ouiaId?: number | string;
   /** Set the value of data-ouia-safe. Only set to true when the component is in a static state, i.e. no animations are occurring. At all other times, this value must be false. */
   ouiaSafe?: boolean;
+  /** @beta The variant of a non-checkbox menu. Determines whether one or more items can be selected. */
+  selectVariant?: 'single' | 'multi';
```

Comment:
Can the description mention specifically that the role and aria props are being manipulated by this prop?  Maybe I'm thinking too much of the original select but currently it sounds like to me that the prop should be changing functionality / handling of selection when it doesn't.

Everything else lgtm.

Reply:
Yes agree with Katie.  The prop name is a little misleading. I am trying to think i there is a better name...

Reply:
Yeah the prop name I wasn't totally sure on. `ariaSelection` could maybe fit a little more, but there are non ARIA attributes being set. selectionType is another one, though may not be better than selectVariant.

I can push an update to the description and see if that would help a bit, though definitely open to updating the prop name as well if we can land on one that's more fitting.

Reply:
@tlabaj @kmcfaul what about, "Determines the roles and ARIA attributes for a non-checkbox menu. Only pass in if one or more menu items can be selected." as a description?

Reply:
Sounds good for the description. Name-wise I haven't been able to think of a better fitting name. `roleAriaType` maybe, but that's a bit unclear still. `roleAndAriaType`?

Reply:
Maybe using `aria-multiselectable` as the prop for Menu and having to pass in true or false? If the prop is undefined,  the Menu isn't a non-checkbox select menu, otherwise true or false could still set the role and ARIA attributes, with the actual boolean setting the `aria-multiselectable` attribute itself. Or maybe `ariaSelectionType` to keep the `"single" | "multi"` typing.

Reply:
That could work... For role we could have it be to set the MenuList role, so either listbox or menu, with that determining the role of MenuItem (if `role="listbox"` then MenuItem would have `role="option"` set).;

@tlabaj wdyt?

Reply:
Yes, I think that would be much more intuitive.  Good idea.

Reply:
@kmcfaul @tlabaj updated! Had to also add a role prop for Select Next so that a checkbox select could have `role="menu"` set on it (with the default being `role="listbox"`)

@nicolethoen would you mind taking another pass in VO?